src/test/java/org/gridsuite/filter/expertfilter/NumberExpertRuleTest.java

```diff
@@ -2383,4 +2388,204 @@ private static Stream<Arguments> provideArgumentsForStaticVarCompensatorTest() {
         );
     }
 
+    private static Stream<Arguments> provideArgumentsForHvdcLinesTest() {
```

Comment:
I don't see the test for nominalV of hdvc line, that's why test coverage is not 100%